internal/kafka/internal/workers/kafka_mgrs/preparing_kafkas_mgr.go

```diff
@@ -24,15 +24,13 @@ type PreparingKafkaManager struct {
 	kafkaService services.KafkaService
 }
 
-// NewPreparingKafkaManager creates a new kafka manager
-func NewPreparingKafkaManager(kafkaService services.KafkaService, bus signalbus.SignalBus) *PreparingKafkaManager {
+// NewPreparingKafkaManager creates a new kafka manager to reconcile accepted kafkas
+func NewPreparingKafkaManager(kafkaService services.KafkaService, bus signalbus.SignalBus, reconciler workers.Reconciler) *PreparingKafkaManager {
```

Comment:
```suggestion
func NewPreparingKafkaManager(kafkaService services.KafkaService, reconciler workers.Reconciler) *PreparingKafkaManager {
```
signal bus argument is not used. It can be removed